tracer/src/Datadog.Trace/Configuration/ImmutableAzureAppServiceSettings.cs

```diff
@@ -57,6 +57,8 @@ public ImmutableAzureAppServiceSettings(IConfigurationSource? source, IConfigura
             FunctionsWorkerRuntime = config.WithKeys(ConfigurationKeys.AzureAppService.FunctionsWorkerRuntimeKey).AsString();
             FunctionsExtensionVersion = config.WithKeys(ConfigurationKeys.AzureAppService.FunctionsExtensionVersionKey).AsString();
 
+            WebsiteSKU = config.WithKeys(ConfigurationKeys.AzureAppService.WebsiteSKU).AsString("Dynamic");
```

Comment:
I'm not sure this logic is correct? 🤔 This will default _all_ applications to assuming they're a Functions app consumption plan, because if the environment variable is not set, we mark `IsFunctionsAppConsumptionPlan=true` 🤔

Reply:
updated!